LPS-93681 Adding test

4lejandrito opened the pull request.

4lejandrito committed:
LPS-93681 Adding isApproved condition to the view

4lejandrito committed:
LPS-93681 Adding condition to the service

4lejandrito committed:
LPS-93681 Adding test

liferay-continuous-integration commented:
CI is automatically triggering &quot;ci:test:sf&quot; and &quot;ci:test:relevant&quot; for this pull to run Source Formatter and relevant tests.

Comment &quot;ci:test&quot; to run the full PR Tester for this pull.

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest(master)/4875/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-3.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4875/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-18.liferay.com/job/test-portal-source-format/723/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-18.liferay.com//userContent/jobs/test-portal-source-format/builds/723/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 2 minutes 27 seconds 957 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8686a0769e574ca304d9662f04a484a3976b4462">8686a0769e574ca304d9662f04a484a3976b4462</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/4lejandrito/liferay-portal/tree/LPS-93681">LPS-93681</a><br/>Branch GIT ID: <a href="https://github.com/4lejandrito/liferay-portal/commit/a00b22ad9c3a079e35736af471474cffeec456a5">a00b22ad9c3a079e35736af471474cffeec456a5</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-18.liferay.com/job/test-portal-source-format/723/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-18.liferay.com/userContent/jobs/test-portal-source-format/builds/723/jenkins-report.html">here</a>.</h5></details></html>

4lejandrito committed:
LPS-93681 Adding support for the triggerTitle attribute

https://github.com/liferay/clay/blob/2.x-develop/packages/clay-dropdown/src/ClayDropdown.soy#L28

4lejandrito committed:
LPS-93681 Label is not for the title. Using triggerTitle instead

liferay-continuous-integration commented:
Closing and locking pull request because pull requests sent to this user may not be updated. Please resend this pull request.

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 22 out of 24 jobs passed in 1 hour 41 minutes 46 seconds 523 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/8686a0769e574ca304d9662f04a484a3976b4462">8686a0769e574ca304d9662f04a484a3976b4462</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/d90ef69f26af27af1d6004abb47d55d370b2c705">d90ef69f26af27af1d6004abb47d55d370b2c705</a></p>22 out of 24 jobs PASSED<div><h4>2 Failed Jobs:</h4><ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest(master)/4875/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305794/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</strong></strike></a></li></ul></div><details><summary><strong>22 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305787/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/285409/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/259681/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43646/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305793/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/293150/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/200596/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157321/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/293151/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/200597/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43651/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157322/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/43652/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/293152/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288408/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/200598/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/157323/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305795/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/290888/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/290889/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/288409/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/284201/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-3.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4875/jenkins-report.html">here</a>.</h5><hr/><h4>This pull contains no unique failures.</h4><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">acceptance upstream results</a> at 8686a0769e574ca304d9662f04a484a3976b4462:</strong></summary><ol><li><div><h5><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/305794/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/305794//consoleText">AXIS_VARIABLE=0,label_exp=!master #305794</a><pre><code>     [exec] * What went wrong:
     [exec] Execution failed for task ':apps:analytics:analytics-client-js:npmRunTest'.
     [exec] &gt; java.io.IOException: Process '[/opt/dev/projects/github/liferay-portal/build/node/bin/node, /opt/dev/projects/github/liferay-portal/modules/yarn-1.13.0.js, --production, false, --registry, http://mirrors.lax.liferay.com:4873, run, test]' finished with non-zero exit value 1
     [exec] 
     [exec] * Try:
     [exec] Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Exception is:
     [exec] org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':apps:analytics:analytics-client-js:npmRunTest'.
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
     [exec] 	at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
     [exec] 	at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
     [exec] 	at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
     [exec] 	at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
     [exec] 	at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:59)
     [exec] 	at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
     [exec] 	at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)</code></pre></div></li></ol></div></li></ol></details></details></html>
